templates/common/baremetal/files/baremetal-mdns-publisher.yaml

```diff
@@ -45,9 +44,9 @@ contents:
               hostname
             fi
           }
-          while [ "$(get_hostname)" == "$DEFAULT_LOCAL_HOSTNAME" ]
+          while [ "$(get_hostname)" =~ ^localhost(.localdomain)?$ ]
```

Comment:
I think you're going to need [[ ]] around this because regexes are a bash thing. If I try this code with single brackets I get: syntax error near unexpected token `('.

However, with double brackets it seems to work and does the right thing.

Reply:
Right. Thanks!

Reply:
This happens to me for testing separately and not paying enough attention on copy paste :-1: